Update docs for ISP and PRS (#930)
vitessio/website: content/en/docs/13.0/user-guides/configuration-advanced/reparenting.md

In this scenario, the old primary's tablet type transitions to `spare`. If health checking is enabled on the old primary, it will likely rejoin the cluster as a replica on the next health check. To enable health checking, set the `target_tablet_type` parameter when starting a tablet. That parameter indicates what type of tablet that tablet tries to be when healthy. When it is not healthy, the tablet type changes to spare.

This command performs the following actions when used to initialize the first primary in the shard:
1. Promote the new primary that is specified.
2. Ensures replication is functioning properly via the following steps:

>     - On the primary-elect tablet, insert an entry in a test table and then update the global Shard object's PrimaryAlias record.

Reviewer: Should 'Shard' here be capitalized?

Reviewer 2: not required.

Reviewer 2: @GuptaManan100 this isn't a "test" table, it's the reparent_journal table. Correct?
We should replace test with internal.

Author: Fixed

>     - On the primary-elect tablet, insert an entry in a test table and then update the global Shard object's PrimaryAlias record.

Reviewer 2: ```suggestion
    - On the primary-elect tablet, insert a row into an internal table and then update the global Shard object's PrimaryAlias record.
```

Author: Fixed

>     - In parallel on each replica, set the new primary and wait for the test entry to replicate to the replica tablet.

Reviewer 2: ```suggestion
    - In parallel on each replica, set the new primary and wait for the inserted row to replicate to the replica tablet.
```

Author: Fixed
